turtlesim/src/turtle_frame.cpp

```diff
@@ -31,38 +31,41 @@
 
 #include <QPointF>
 
-#include <ros/package.h>
 #include <cstdlib>
 #include <ctime>
 
 #define DEFAULT_BG_R 0x45
 #define DEFAULT_BG_G 0x56
 #define DEFAULT_BG_B 0xff
 
+using namespace std::chrono_literals;
+
 namespace turtlesim
 {
 
-TurtleFrame::TurtleFrame(QWidget* parent, Qt::WindowFlags f)
-: QFrame(parent, f)
-, path_image_(500, 500, QImage::Format_ARGB32)
-, path_painter_(&path_image_)
-, frame_count_(0)
-, id_counter_(0)
+TurtleFrame::TurtleFrame(rclcpp::Node::SharedPtr &node_handle, QWidget* parent, Qt::WindowFlags f)
+: QFrame(parent, f),
+  path_image_(500, 500, QImage::Format_ARGB32),
+  path_painter_(&path_image_),
+  frame_count_(0),
+  id_counter_(0)
 {
   setFixedSize(500, 500);
   setWindowTitle("TurtleSim");
 
   srand(time(NULL));
 
   update_timer_ = new QTimer(this);
-  update_timer_->setInterval(16);
+  update_timer_->setInterval(16); // msec
```

Comment:
Since this is unrelated to ROS 2 please consider contributing this against the default branch instead.

Reply:
I've deleted `// msec`